Run/EnergyPositionHistosRun.cpp

```diff
@@ -8,10 +11,39 @@ int main() {
     auto [world, names, gdmlPath] = LUXEGeometry::MockupGeant4World();
 
     Acts::GeometryContext gctx;
+    std::string gdmlPath2 = "lxgeomdump_staves.gdml";
+    std::vector<std::string> names2{"OPPPSensitive"};
 
     // Build the LUXE detector
     auto detector =
-        LUXEGeometry::buildLUXEDetector(gdmlPath, names, gctx);
+        LUXEGeometry::buildLUXEDetector(gdmlPath2, names2, gctx);
+
+
+    // map (x,y,z) -> (x,y,z)
+    auto transformPos = [](const Acts::Vector3& pos) {
+        return pos;
+    };
+
+    // map (Bx,By,Bz) -> (Bx,By,Bz)
+    auto transformBField = [](const Acts::Vector3& field, const Acts::Vector3&) {
+        return field;
+    };
+
+    auto bFieldValue = [](const std::array<double, 3> &v) {
```

Comment:
Let's keep it inside the `LUXEMagneticField`. That way we can have several predefined fields that we'll call inside the Run